sdk/identity/identity/review/identity.api.md

```diff
@@ -34,10 +34,11 @@ export const AuthenticationErrorName = "AuthenticationError";
 // @public
 export interface AuthenticationRecord {
     authority: string;
+    clientId: string;
     homeAccountId: string;
-    serialize: () => string;
     tenantId: string;
     username: string;
+    version: SupportedAuthenticationRecordVersions;
```

Comment:
We don't want to add version to the properties of this interface, only to the serialized content. We can expose it as a property if we ever add more than a single version, but until then it's not something the user needs to consider.

Reply:
Thank you Scott!